.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Describe the changes made and why they were made instead of how they were made.
+
+## Related issues and discussion
+#{Issue Number}
+ 
+
+## Checklist
+ 
+Make sure you've done all the following (You can delete the checklist before submitting)
+ 
+ - [ ] Code is well documented
+ - [ ] Include unit tests for new functionality
+ - [ ] Code passes Travis builds in your branch
+ - [ ] If you have multiple commits please combine them into one commit by squashing them.
+ - [ ] Read and understood the contribution guidelines .
```

Comment:
Should this link to the contribution guidelines?

Reply:
Yes I think we should link it to the contribution guidelines.